components/automate-ui/src/app/modules/desktop/top-errors/top-errors.component.html

```diff
@@ -18,7 +18,7 @@ <h6 class="heading-time">Updated {{ lastUpdatedMessage }}</h6>
         (click)="errorSelected.emit(item)"
         (keyup.enter)="errorSelected.emit(item)">
         <chef-td class="chart-list-item-label-cell">
-          {{ item.type }}: {{ item.message }}
+          <div>{{ item.type }}: {{ item.message }}</div>
```

Comment:
We can't easily add ellipses to a table cell without changing it's display, i.e. `display: block`, so instead we can wrap the text in a div and apply the necessary css there.